DOL.WHD.Section14c.Web/src/v4/ui-library.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ui-library',
+  templateUrl: 'src/v4/ui-library.component.html',
+  styleUrls: ['src/v4/ui-library.component.css']
```

Comment:
Not sure how much it matters, but these URLs fail to resolve in the dist build.  Everything works great with the webpack-dev-server (e.g., `npm start`), but using the static output from `npm run deploy`, those files aren't there.  Looks like there's an [Angular template loader for Webpack](https://www.npmjs.com/package/angular2-template-loader) for pretty much exactly this.  (Also [a useful StackOverflow question](https://stackoverflow.com/questions/39157580/angular2webpack-how-should-i-output-html-file-with-templateurl) about the same thing.)

Another question for @mmurthydol.  Should the deployable static files include this test page?  If not, there's nothing to do here.  If so, the build process needs to incorporate template files.

Reply:
@brendansudol @mgwalker : Thanks for the conversation. The 508 test page will be tested by DOL/WHD's accessibility tester's from the DEV/TEST environments. So basically we need this page available in lower environment builds (not in Staging/Production).  Wondering what our EZ options are based on this requirement?

Reply:
Make this page build as part of the builds  and remember to remove the page it from higher environment code base/builds? Use a fancy UUID query string to allow display of this page? Any others? Thanks again for the thoughts; we'd want to document what we decide to do so it could become part of a 14c operations manual.

Reply:
The way the build works, you can't really remove pages after the fact.  I think the easiest thing is to have this page included in the build.  The change to make that happen will probably need to be made anyway, to support new components that the vendor will build.  I also don't see a reason to make the build more complicated by trying to exclude certain parts based on some condition, so I'd say just leave this page in the production build.  It's harmless.

Reply:
@mgwalker That sounds right Greg :)  I didn't want to muck around with different build steps per environment. I was really thinking - If business really has a problem with it, I was just going to replace that page content with harmless 404 content in the production branch.